Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
Try rebuilding again. It looks like your build ran a restore while our CI was pushing an update. The latest version of Microsoft.AspNetCore.Mvc.Razor.ViewCompilation is 2.0.0-preview3-26025